### ISSUE TITLE
-nofit option for gmx_cluster

### ISSUE THREAD

**Issue by PabloNA97** [bioexcel/biobb_analysis#20]
The same group is used for trajectory aligning and RMSD calculation in gmx cluster (as far as I know). Thus, if one wants to align with respect to one group (e.g. all backbone heavy atoms) and compute the RMSD with respect to another group (e.g. active site residues) the only possibility is to align the trajectory before applying gmx_cluster (for example with gmx trjconv through gmx_image) and then apply gmx cluster with the -nofit option activated. Which is not available through gmx_cluster yet. Thank you! :)

**Comment by PauAndrio** [comment]
Hi!

Please, could you paste here the specific Gromacs command line here. I have added the -nofit flag property however gmx cluster still only asks for just one group for both: fitting and RMSD calculation.

```
Command line:
  gmx cluster -g gmx_cluster_cluster.log -dist gmx_cluster_rmsd-dist.xvg -o gmx_cluster_rmsd-clust.xpm -s /private/tmp/biobb/unitests_2/gmx_cluster/5c8d32ce-0b55-45c1-a725-8e223c5c7c24/topology.tpr -f /private/tmp/biobb/unitests_2/gmx_cluster/5c8d32ce-0b55-45c1-a725-8e223c5c7c24/trajectory.trr -cl /private/tmp/biobb/unitests_2/gmx_cluster/5c8d32ce-0b55-45c1-a725-8e223c5c7c24/output_pdb.pdb -cutoff 0.1 -method linkage -n /private/tmp/biobb/unitests_2/gmx_cluster/5c8d32ce-0b55-45c1-a725-8e223c5c7c24/index.ndx -nofit

Select group for least squares fit and RMSD calculation:
Group     0 (         System) has 29509 elements
Group     1 (            DNA) has   758 elements
Select a group: 0
Selected 0: 'System'

Select group for output:
Group     0 (         System) has 29509 elements
Group     1 (            DNA) has   758 elements
Select a group: 0
Selected 0: 'System'
```

**Comment by PabloNA97** [comment]
I think that is correct, you are not missing any flag as far as I know.
The -nofit flag should deactivate the fitting part even though the group is still called "least squares fit and RMSD calculation". 
With this option available now we can fit the trajectory beforehand using trjconv and then compute the RMSD with -nofit. Like this we can fit with respect to one group and compute the RMSD with respect to another.

**Comment by PauAndrio** [comment]
Solved in [5347ca5](https://github.com/bioexcel/biobb_analysis/commit/5347ca57a6f2a417e23b4403d40da964f5e4b7b1)
Pablo, let me know either if you need a new release or you can use the development version.
